### PR TITLE
feat(gastro): metatdata

### PR DESCRIPTION
**Pull request by alghanor**
origin object on deltas and on transactions

### CODE REVIEW

File: lib/v1/gastro_orders/deltas/base.js [hunk 1]
```diff
@@ -71,30 +71,35 @@ module.exports = {
...
+          room: oneOf({
+            type: 'string',
+            format: 'uuid',
+            example: '00d2351d-ea67-05f9-ad4f-0abcbefa903a',
+            description: 'Id of the most recent gastro room that this process belongs to'
+          }),
+          room_name: oneOf({
+            type: 'string',
+            maxLength: 128,
+            example: 'Table 23',
```
Comment (boris-savic) [inline lib/v1/gastro_orders/deltas/base.js:98]: Change the example text to "Garden"

File: lib/v1/transactions/legacy/base.js [hunk 2]
```diff
@@ -208,6 +208,35 @@ module.exports = {
...
+                room: oneOf({
+                  type: 'string',
+                  format: 'uuid',
+                  example: '00d2351d-ea67-05f9-ad4f-0abcbefa903a',
+                  description: 'Id of the most recent gastro room that this process belongs to'
+                }),
+                room_name: oneOf({
+                  type: 'string',
+                  maxLength: 128,
+                  example: 'Table 23',
```
Comment (boris-savic) [inline lib/v1/transactions/legacy/base.js:236]: Same here. Change room name example to "Garden"

File: lib/v1/transactions/legacy/base.js [hunk 3]
```diff
@@ -208,6 +208,35 @@ module.exports = {
               format: 'date-time',
               example: '2019-01-23 16:51:29.143+00',
               description: 'Starting time of the first delta'
+            },
+            origin: {
+              type: 'object',
+              properties: {
+                table: oneOf({
+                  type: 'string',
```
Comment (aleksey-a-maltsev) [inline lib/v1/transactions/legacy/base.js:216]: May be better to make this properties set reusable to avoid copypaste?

Comment (alghanor) [reply lib/v1/transactions/legacy/base.js:216]: yes

**Review by aleksey-a-maltsev — APPROVED** [review]

**Pull request merged**